squirrels/_command_line.py

```
@@ -137,12 +142,27 @@ def with_file_format_options(parser: ArgumentParser):
+        '--forwarded-allow-ips', type=str, default=None, 
+        help="Comma separated list of IP Addresses, IP Networks, or literals (e.g. UNIX Socket path) to trust with proxy headers. "
+            "Defaults to the FORWARDED_ALLOW_IPS environment variable if available, or '127.0.0.1'. The literal '*' means trust everything."
+    )
 
     args, _ = parser.parse_known_args()
     
+    update_checker_result = UpdateChecker().check("squirrels", __version__)
+    if update_checker_result:
+        current_version = update_checker_result.running_version
+        latest_version = update_checker_result.available_version
+        print(f"\033[1;33mA newer version of Squirrels is available!\033[0m")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)